### PR TITLE
Adjusts Cargo Market

### PR DESCRIPTION
**Pull request by BlueHNT**
# Description

Long story short, cargo and command abuses the new reverse-engineered suits and warning them as admins about it is getting quite tiring. So I decided to put my hand to the craft and comment them out for now, until a better way for them to be gotten is made.

I have also taken this opportunity to fix some of the longstanding issues with cargo I have had, such as there missing sec masks in the riot crate or there not being a way to get the swat items.

While I am still missing the properly developed versions with space proofing and minor cold/head protection, the old non ARCS riot suit will do for now.

This PR fixes https://github.com/Simple-Station/Einstein-Engines/issues/1376

---

# TODO

- [x] Remove problematic items from cargo
- [x] Add proper swat crate

Opt:
- [x] Make proper swat set (soon:tm:)
- [x] Adjust the armor values to fit better (can be delayed)

---

<details><summary><h1>Media</h1></summary>
<p>

I ain't got nothing chief.

</p>
</details>

---

# Changelog

:cl:
- add: Added swat suit
- add: Added swat crate
- add: Added new crate category for secure reinforced crates
- tweak: Tweaked security supplies and crates
- tweak: Tweaked the price of Cybersun's tacsuits because apparently they weren't charging enough to properly scam salvage techs.
- tweak: Tweaked `StructuralMetallicStrong` structural protection from 10 to 80

### CODE REVIEW

**Comment by sleepyyapril** [comment]
test fail real

**Review by VMSolidus — APPROVED** [review]

**Comment by VMSolidus** [comment]
Test fails are legitimate.

**Comment by BlueHNT** [comment]
I'm unable to find the issue.

File: Resources/Prototypes/Catalog/Fills/Crates/security.yml [hunk 1]
```diff
@@ -41,22 +41,43 @@
...
+  name: riot crate
+  description: Contains two sets of riot armor, helmets, shields, and enforcers loaded with beanbags. Extra ammo is included. Requires Security access to open.
   components:
   - type: StorageFill
     contents:
       - id: ClothingOuterArmorARC # DeltaV - ClothingOuterArmorRiot replaced in favour of ARC suit
         amount: 2
       - id: ClothingHeadHelmetRiot
         amount: 2
+      - id: Clothing
```
Comment (VMSolidus) [inline Resources/Prototypes/Catalog/Fills/Crates/security.yml:53]:
```suggestion
```

**Review by VMSolidus — CHANGES_REQUESTED** [review]

**Review by VMSolidus — APPROVED** [review]
It was a stray "-type: Clothing" in a place where there shouldn't have been one.

**Comment by BlueHNT** [comment]
Yeah that 100% shouldn't have been there. No wonder I couldn't find it.

File: Resources/Prototypes/Damage/modifier_sets.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@
     Slash: 10
     Piercing: 10
     Heat: 10
-    Structural: 10
+    Structural: 30
```
Comment (VMSolidus) [inline Resources/Prototypes/Damage/modifier_sets.yml:25]:
```suggestion
    Structural: 80
```
image
Pickaxe actually does 52.5 damage on a wide swing, and if you stack modifiers you can get it up to 78.75 structural. So to make it pickaxe-proof, it needs to be MUCH higher.

**Review by VMSolidus — CHANGES_REQUESTED** [review]

**Review by OldDanceJacket — APPROVED** [review]

**Review by VMSolidus — APPROVED** [review]

**Pull request merged**